README.md

```diff
@@ -112,7 +112,7 @@ algorithm = GroundStateEigensolver(mapper, vqe)
 electronic_structure_result = algorithm.solve(problem)
 print(electronic_structure_result)
 ```
-The program above uses a quantum computer to calculate the ground state energy of molecular Hydrogen,
+The program above uses a virtual quantum computer simulator to calculate the ground state energy of molecular Hydrogen,
```

Comment:
```suggestion
The program above computes the ground state energy of molecular Hydrogen,
```
The text there, which includes `uses a quantum computer`, has been around a long time. In the past the backend was arguably always more evident, see here https://github.com/Qiskit/qiskit-aqua#creating-your-first-chemistry-programming-experiment-in-qiskit . Qiskit is now using primitives, and in this case no it's not using Aer simulator, rather its done using the reference Estimator from qiskit.primitives which is implemented using the quantum_info module in Terra. Nowadays, pointing out that this is quantum, which was done from when this sample code first came about, I would argue is less needed. And rather than worry about where its computed, which can simply be changed by changing the Estimator, I would more say that if any change is needed here that I would more just drop that now from the text altogether as per my suggestion here.